src/cogs/misc/main.py

```diff
@@ -131,25 +132,34 @@ async def _uptime(self, ctx: commands.Context):
         await ctx.send(embed=em)
 
     @commands.hybrid_command(name="poll", aliases=["ynpoll", "pollstart"])
-    async def _ynpoll(self, ctx: commands.Context, *, question: str):
+    async def _ynpoll(self, ctx: commands.Context, *, args: str):
```

Comment:
A better way to do this is with typing.Optional[], rather than handling the args within the command.
```python
async def _ynpoll(
        self, ctx: commands.Context, time: typing.Optional[int] = 30, *, question: str
    ):